app/controllers/user_organization_memberships_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UserOrganizationMembershipsController < ApplicationController
+  before_action :ensure_logged_in
+  before_action :ensure_member, only: :organization_memberships
```

Comment:
This seems quite specific to me, i'd just have this as the first line in the organization_memberships action